docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,101 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
```

Comment:
@medubelko as we discussed earlier, I ended up putting everything that I would put on a changelog here. Please take a look and point out the items you feel are out-of-place here.